### PR TITLE
Disable SLO alerts except for adminapi/apiserver/server.

### PR DESCRIPTION
**Pull request by yegle**
Add a knob to set latency SLO threshold.

### CODE REVIEW

File: terraform/alerting/module.availability-slo/variables.tf [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
-  description = "Whether to enable this availability SLO."
-}
-
 variable "goal" {
   type        = number
   description = "Availability SLO goal."
 }
 
-variable "notification-channels" {
+variable "notification_channels" {
```
Comment (sethvargo) [inline terraform/alerting/module.availability-slo/variables.tf:14]: This will require updates to all the projects in -infra

**Review by sethvargo — APPROVED** [review]
/lgtm

**Review by mikehelmick — COMMENTED** [review]
let's include enx-redirect as well

File: terraform/alerting/module.availability-slo/variables.tf (hunk 1 above)
Comment (yegle) [reply terraform/alerting/module.availability-slo/variables.tf]: This is a module used by en-alerting only, no need to change the infra repo.

**Comment by yegle** [comment]
> let's include enx-redirect as well

enx-redirect will have alert disabled.

**Comment by yegle** [comment]
cc @mariliamelo @yuriatgoogle

File: terraform/alerting/module.availability-slo/main.tf [hunk 2]
```diff
@@ -22,6 +22,7 @@ resource "google_monitoring_slo" "slo" {
   slo_id       = "availability-slo-${var.service_name}"
   display_name = "${var.goal * 100}% of requests are successful over rolling 28 days (service=${var.service_name})"
   project      = var.project
+  count        = var.enabled ? 1 : 0
```
Comment (sethvargo) [inline terraform/alerting/module.availability-slo/main.tf]: Nit, count typically comes at the very top of the resource.

Comment (yegle) [reply terraform/alerting/module.availability-slo/main.tf]: Fixed.

**Review by yuriatgoogle — APPROVED** [review]

**Pull request merged**